Sources/Sentry/SentryFramesTrackingIntegration.m

```diff
@@ -0,0 +1,45 @@
+#import "SentryFramesTrackingIntegration.h"
+#import "SentryDisplayLinkWrapper.h"
+#import "SentryFramesTracker.h"
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface
+SentryFramesTrackingIntegration ()
+
+#if SENTRY_HAS_UIKIT
+@property (nonatomic, strong) SentryFramesTracker *tracker;
+#endif
+
+@end
+
+@implementation SentryFramesTrackingIntegration
+
+- (void)installWithOptions:(SentryOptions *)options
+{
+#if SENTRY_HAS_UIKIT
+    if (options.enableFrameRenderMeasuring) {
+        self.tracker = [SentryFramesTracker sharedInstance];
+        [self.tracker start];
+    }
```

Comment:
l: should we log if it's disabled or no `SENTRY_HAS_UIKIT`? we always do it on Java